### DISCUSSION TITLE
How to edit footer copyright?

### DISCUSSION THREAD

**Discussion by szarkans** [squidfunk/mkdocs-material#7068] [Q&A] [answered]
Hi! I want to edit "Made with Material for MkDocs" with "Made with love and Material for MkDocs" but in my language. I tried to add `override` and extend the footer, but my pages start to duplicate for some reason...

**Comment by alexvoss** [comment, marked as answer]
The generator message is in the `copyright.html` partial.  Copy it to your `overrides/partials ` directory and change the text there. In `mkdocs.yml` you should have:

```yaml
theme:
  name: material
  custom_dir: overrides
```

Not sure what you mean by "my pages start to duplicate"?

**Reply by szarkans** [reply]
> my pages start to duplicate
![image](https://github.com/squidfunk/mkdocs-material/assets/150809885/4ccb66ee-7e17-4bd3-8644-7bc5ea914a42)


Code is 
```html
{% extends "base.html" %}

{% block footer %}

<div class="md-copyright">
    Сделано с любовью и <a href="https://github.com/squidfunk/mkdocs-material">Material for MkDocs</a>
</div>

{% endblock %}
```

Also - could you please point where i can find copyright.html? I don't understand...

**Reply by szarkans** [reply]
Nevermind - found `copyright.html` in this repo. Thanks!

![image](https://github.com/squidfunk/mkdocs-material/assets/150809885/bb0f98d9-11f2-4773-9d5f-4bf64dc45588)
